birdhouse/components/stac/service-config.json.template

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DACCS-Climate/Marble-node-registry/main/node_registry.schema.json#service",
+  "name": "stac",
+  "keywords": [
+    "catalog"
+  ],
+  "description": "STAC catalog for discovering data on the Marble Network.",
+  "links": [
+    {
+      "rel": "service",
+      "type": "text/html",
+      "href": "https://${PAVICS_FQDN_PUBLIC}/stac/"
+    },
+    {
+      "rel": "service-doc",
+      "type": "text/html",
+      "href": "https://stacspec.org/en"
```

Comment:
> https://raw.githubusercontent.com/radiantearth/stac-api-spec/main/core/openapi.yaml ?

Wouldn't that be better for a `service-desc` link than a `service-doc` link?

Reply:
Yes. 
Just need to be careful about which links are provided. "STAC Spec" refers to the contents of a STAC document, while "STAC-API Spec" are for the service.

Even then, it's somewhat complicated because there are multiple OpenAPI for core, item-seach and ogc-api-features extensions.

Anyway, I think for the JSON structure of the API, the following would be the best we can do:

```json
{
  "rel": "service-doc",
  "type": "text/html",
  "href": "https://github.com/radiantearth/stac-api-spec"
},
{
  "rel": "service-desc",
  "type": "application/yaml",
  "href": "https://raw.githubusercontent.com/radiantearth/stac-api-spec/main/core/openapi.yaml"
}
```